_posts/2020-07-31-kubeflow-1.1-blog-post.md

```diff
@@ -0,0 +1,39 @@
+
+The isolation and security feature deliveries include Private GKE and Anthos support, a stable version of Kubeflow Pipelines with [Multi-User Kubeflow Pipelines](https://github.com/kubeflow/pipelines/projects/1) support, and a process for Kubeflow container image scanning, [CVE reporting](https://docs.google.com/spreadsheets/d/1ijWIyjGQpDy68-vjBmLyFY0U1oGCT--A0nVGgzLGDhU/edit#gid=0), and an optional process for distroless image creation. 1.1 also includes [options for authentication and authorization](https://github.com/kubeflow/kubeflow/issues/4960). This includes the option for administrators to turn off self-service namespace creation mode, as admins may have other processes for namespace creation. The Community also developed a best practice to build user authorization in Kubeflow web apps using subject access review.
+
+The installation and operations of Kubeflow have been enhanced to support GitOps methodologies. Several Kubeflow platform providers and software support vendors are developing time-saving GitOps processes to simplify and codify the installation, configuration and operations of the various layers in the Kubeflow 1.1 hardware and software stack. Some examples are provided in the next section.
+
+**More details and 1.1 tutorials**
+
+Kubeflow 1.1 includes many technical enhancements, which are being delivered via the Community's release process. Details on the application feature development can be found in the [1.1 KanBan Board](https://github.com/orgs/kubeflow/projects/26) and in the [Kubeflow Roadmap](https://github.com/kubeflow/kubeflow/blob/master/ROADMAP.md). As the Kubeflow application improvements are merged, the platform teams (GCP, AWS, IBM, Red Hat, Azure, and Arrikto MiniKF) are working to validate the feature improvements on their respective environments.
+
+Kubeflow 1.1 demo scripts and workflow tutorials are available as validated by the individual platforms. Please find those below:
+
+- [Kubeflow 1.1 Tutorial for Automated Hyperparameter Tuning, Multi-user Pipelines, Pipeline Caching](https://medium.com/@sfiora/5821747f4fcb)
+- [GitOps for Kubeflow 1.1 on AWS EKS](http://docs.arrikto.com/integrations/kubeflow.html)
+
+1.1 users can also leverage several other Kubeflow EcoSystem tools including:
+
+- [Seldon Core 1.1](https://docs.seldon.io/projects/seldon-core/en/latest/reference/release-1.1.0.html), which handles scaling to thousands of production ML models and provides advanced ML capabilities including Advanced Metrics, Request Logging, Explainers, Outlier Detectors, A/B Tests, Canaries and more.
+- The [Feast](https://feast.dev/) 0.3 feature store, which allows ML teams to define, manage, discover, and serve ML features.
```

Comment:
How about
```
[Feast](https://feast.dev/): A feature store that allows teams ML teams to define, manage, discover, and serve ML features to their models.
```
Not sure how important the version is, and it will go out of date quickly anyways.